[2025-09-02T02:32:10.119+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-02T02:32:10.280+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_pipeline.top_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T02:32:10.307+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_pipeline.top_products scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T02:32:10.309+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-02T02:32:10.362+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): top_products> on 2025-09-01 00:00:00+00:00
[2025-09-02T02:32:10.370+0000] {standard_task_runner.py:63} INFO - Started process 610 to run task
[2025-09-02T02:32:10.382+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_pipeline', 'top_products', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '160', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_pipeline_dag.py', '--cfg-path', '/tmp/tmp1f0g211l']
[2025-09-02T02:32:10.385+0000] {standard_task_runner.py:91} INFO - Job 160: Subtask top_products
[2025-09-02T02:32:10.425+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-02T02:32:10.577+0000] {task_command.py:426} INFO - Running <TaskInstance: ecommerce_pipeline.top_products scheduled__2025-09-01T00:00:00+00:00 [running]> on host 6aba05baf296
[2025-09-02T02:32:10.865+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='arel' AIRFLOW_CTX_DAG_ID='ecommerce_pipeline' AIRFLOW_CTX_TASK_ID='top_products' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-09-02T02:32:10.867+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-02T02:32:10.915+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-02T02:32:10.918+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'PGPASSWORD=*** psql -h postgres -U *** -d *** -f /opt/***/sql/top_products.sql']
[2025-09-02T02:32:10.972+0000] {subprocess.py:86} INFO - Output:
[2025-09-02T02:32:11.506+0000] {subprocess.py:93} INFO - psql:/opt/***/sql/top_products.sql:1: NOTICE:  table "top_products" does not exist, skipping
[2025-09-02T02:32:11.507+0000] {subprocess.py:93} INFO - DROP TABLE
[2025-09-02T02:32:11.512+0000] {subprocess.py:93} INFO - psql:/opt/***/sql/top_products.sql:12: ERROR:  relation "transactions" does not exist
[2025-09-02T02:32:11.516+0000] {subprocess.py:93} INFO - LINE 7: FROM transactions
[2025-09-02T02:32:11.517+0000] {subprocess.py:93} INFO -              ^
[2025-09-02T02:32:11.551+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-02T02:32:11.555+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-02T02:32:11.699+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ecommerce_pipeline, task_id=top_products, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20250902T023210, end_date=20250902T023211
[2025-09-02T02:32:11.852+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-02T02:32:11.894+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-02T02:32:11.903+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
